### PR TITLE
Generate Land Use data for EDDIE

### PR DESCRIPTION
**Pull request by wsp-mbuckley**
Script to perform some comparisons and pre-processing for EDDIE workbook:
- Read EDDIE land use inputs and compare with NPIER land use data
- Convert NPIER data into format ready for input into EDDIE.

### CODE REVIEW

**Comment by wsp-mbuckley** [comment]
Currently this process outputs NPIER land use in a spreadsheet ready to be copied into EDDIE workbook. In future functionality should be added to have the script update the EDDIE workbook directly.

File: normits_demand/utils/pandas_utils.py [hunk 1]
```diff
@@ -13,10 +13,11 @@
 # Builtins
```
Comment (BenTaylor-TfN) [inline normits_demand/utils/pandas_utils.py:13]: Tagging this file to watch out for. 

I need to get moving on #962 which aims to remove the pandas_utils functions over to caf.toolkit

File: run_helper_scripts/extrapolate_ddgs.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+        return
+
+    calc_range = _check_calculation_years(year_columns, calculation_years)
+
+    yearly_increase = (ddg[calc_range[1]] - ddg[calc_range[0]]) / (
+        calc_range[1] - calc_range[0]
+    )
+    yearly_increase.name = "Average Yearly Increase"
+    increase_path = output_file.with_name(output_file.stem + "-average_yearly_increase.csv")
+    yearly_increase.to_csv(increase_path)
```
Comment (BenTaylor-TfN) [inline run_helper_scripts/extrapolate_ddgs.py:123]: Is this the yearly increase "report" looking thing being output? Could all of these be placed into a subfolder to make it easier to select all the DDGs I want and copy / paste them where I need them?

I've manually done this in the outputs already if you want a point of reference

File: run_checker_scripts/eddie_comparison.py [hunk 3]
```diff
@@ -0,0 +1,1861 @@
+# -*- coding: utf-8 -*-
+"""
```
Comment (BenTaylor-TfN) [inline run_checker_scripts/eddie_comparison.py:2]: There's a lot of code here - let's put aside some time to talk through this code and treat it more as a handover

File: run_checker_scripts/eddie_ddg_comparison.py [hunk 4]
```diff
@@ -0,0 +1,473 @@
+# -*- coding: utf-8 -*-
```
Comment (BenTaylor-TfN) [inline run_checker_scripts/eddie_ddg_comparison.py:1]: Same here, lets chat over this file at some point too

**Comment by wsp-mbuckley** [comment]
@BenTaylor-TfN ready to be merged once your happy with it

**Review by BenTaylor-TfN — APPROVED** [review]

**Pull request merged**